Dependencies/pkgs/onscrlib.pkgbuild

```diff
@@ -3,7 +3,7 @@ pkgver=1.0
 pkgrel=1
 type=meta
 
-depends=(
+depends+=(
```

Comment:
Is it concatenating with something?

Reply:
no, most likely a legacy thing from testing discord